src/screens/auth/SignIn.tsx

```
@@ -51,7 +59,21 @@ const SignIn: React.FC = () => {
 					// Store wallet token if available
 					localStorage.setItem('walletToken', response.data.walletToken);
 				}
-
+				const decoded = jwtDecode(response.data.access_token) as DecodedToken;
```

ESLint

[error] Replace `response.data.access_token` with `⏎↹↹↹↹↹response.data.access_token⏎↹↹↹↹`

(prettier/prettier)